platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1644,6 +1668,45 @@ MGL_EXPORT
  */
 @property (nonatomic, null_resettable) NSExpression *textVariableAnchor;
 
+/**
+ The property allows to control an orientation of a symbol. Note, that the
+ property values act as a hint, so that Symbols whose language doesn't support
+ provided orientation, will be laid out in their natural orientation. Example:
+ English point symbol will be rendered horizontally even if `"textWritingMode":
+ ["vertical"]` is set. The order of elements in an array define priority order
```

Comment:
The code snippet here should be valid Objective-C syntax:

> even if `textWritingMode` is set to `@[@"vertical"]`

The codegen script has some logic to transform JSON-style phrases in the style specification into Objective-C, but it probably doesn’t handle arrays so well. Since this is a bit of an edge case, it’d be fine to override the documentation just for this one property.

Reply:
I changed documentation a bit, if platform specific example is needed, can be added in separate PR.